src/lib/y2storage/storage_manager.rb

```diff
@@ -366,6 +396,37 @@ def reset_staging_revision
       @staging_revision_after_probing = 0
     end
 
+    # Sanitizes the raw probed devicegraph
+    #
+    # @note The raw probed devicegraph can contain some errors (e.g., incomplete LVM VGs).
+    #   This method tries to fix all errors from raw probed.
+    #
+    #   With the default callbacks, the user is asked whether to sanitize the raw probed
+    #   devicegraph when it contains some errors.
+    #
+    #   The raw probed devicegraph remains untouched, and the new sanitized one is internally
+    #   saved and copied into the staging devicegraph.
+    #
+    # @raise [Y2Storage::Error] if the raw probed cannot be sanitized
```

Comment:
Well, to be precise it raises the error if the callbacks (i.e. the user) decided to not sanitize. Is not that we cannot, is that we were told not to do it.

Reply:
Is maybe worth mentioning that in that case `probed` and `staging` remain untouched. They only become useless for installation/partitioner purposes, but they are still there an normally accessible.